src/engine/GroupBy.cpp

```diff
@@ -1253,7 +1254,7 @@ GroupBy::HashMapAggregationData<NUM_GROUP_COLUMNS>::getHashEntries(
   }
 
   auto resizeVectors =
-      []<VectorOfAggregationData T>(
+      []<typename T>(
```

Comment:
II think either `cpp_template_lambda` or at least `QL_CONCEPT_OR_TYPENAME` can be used here.

Reply:
replaced as suggested [here](https://github.com/ad-freiburg/qlever/compare/9dc2786bf8dab4b0810830203ce18222a4fd7d4f..c33b909d36ff52194e77ab5d5c7f528b39b46c31)